app/models/item.rb

```diff
@@ -1,15 +1,31 @@
 class Item < ActiveRecord::Base
 
   scope :active, -> {where(retired:false)}
-  has_attached_file :image, :default_url => "default.jpg"
 
   has_many :items_categories
   has_many :categories, through: :items_categories
-
+  has_many :photos
   has_many :item_orders
   has_many :orders, through: :item_orders
+  belongs_to :tenant
 
   validates :description, length: { in: 0..255, allow_nil: false }, presence: true
   validates :title, presence: true, uniqueness: true, allow_blank: false
-  validates :price, presence: true, :numericality => { :only_integer => true }
+  validates :price, presence: true,
+            :numericality =>
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.